### ISSUE TITLE
How to get base‘s coordinate when mouse hovers over tracks

### ISSUE THREAD

**Issue by BioEvo** [GMOD/jbrowse#1321]
# How to configure JBrowse to display base's coordinate when mouse hovers over a read in a BAM track

when mouse hovers over a BAM track, the blubble will display the rname of reads, it's convenient to use left click to get reads information, but it seems impossible to find out the base coordinate quickly.

**Comment by cmdcolin** [comment]
You can hover over the location bar to get coordinate this way

![image](https://user-images.githubusercontent.com/6511937/53573519-a6428f80-3b3b-11e9-949e-2212bbc6c9dc.png)

I know that igv has some per-base hovering that jbrowse does not have however, is that what you are asking about

**Comment by BioEvo** [comment]
> You can hover over the location bar to get coordinate this way
> 
> ![image](https://user-images.githubusercontent.com/6511937/53573519-a6428f80-3b3b-11e9-949e-2212bbc6c9dc.png)
> 
> I know that igv has some per-base hovering that jbrowse does not have however, is that what you are asking about

Yeah, and location bar works, that's great!  But per-base hovering might be more convenient.
